nf_core/launch.py

```diff
@@ -396,17 +386,17 @@ def prompt_param(self, param_id, param_obj, is_required, answers):
         """Prompt for a single parameter"""
 
         # Print the question
-        question = self.single_param_to_pyinquirer(param_id, param_obj, answers)
-        answer = PyInquirer.prompt([question])
-        # TODO: use raise_keyboard_interrupt=True when PyInquirer 1.0.3 is released
+        question = self.single_param_to_questionary(param_id, param_obj, answers)
+        answer = questionary.prompt([question])
+        # Raise keyboard interrupg
```

Comment:
Can hopefully get ride of this, but until then - typo..

```suggestion
        # Raise KeyboardInterrupt exception
```